build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     // Use JUnit test framework.
     testImplementation 'junit:junit:4.13.2'
     testImplementation 'org.mockito:mockito-core:5.3.1'
+    testImplementation 'com.github.stefanbirkner:system-rules:1.19.0'
```

Comment:
I will need to check if we can include a dependency on a library that isn't MIT or Apache-2 licensed.

Reply:
If it is allowed you will also need to update the `THIRD-PARTY` file

Reply:
Have you considered using https://github.com/stefanbirkner/system-lambda? It is MIT licensed. Also takes advantage of Java 8+ features.

Reply:
Good catch! I used it as well in another project. I am going to switch to system-lambda.

Reply:
I remember why I haven't used it :) it requires JUnit5. For Junit4 we need to use System Rules. Are you ok muve to JUnit5?

Reply:
JUnit5 should be OK. Maybe even preferable.